deployment/src/main/java/io/quarkiverse/quinoa/deployment/framework/override/NextFramework.java

```diff
@@ -31,8 +32,8 @@ public QuinoaConfig override(QuinoaConfig delegate, Optional<JsonObject> package
                     LOG.warn(
                             "Make sure you define  \"build\": \"next build \", in the package.json to make Next work with Quinoa.");
                 }
-                String output = packageJson.get().getString("output");
-                if (!"export".equals(output)) {
+                
+                if (!packageJson.get().containsKey("output") || !OUTPUT.equals(packageJson.get().getString("output"))) {
```

Comment:
AFAIK `packageJson.get().getString("output")` will return null if it's not defined and the check will be false, so what's the difference?

Reply:
Ahh we are using Jakarta JsonObject not Vertx, ok implementation are different it seems, you need to use `getString("output", null)`

Reply:
Ok, I didn't know you can do that either.